### PR TITLE
New package: geany-themes-1.24

### PR DESCRIPTION
**Pull request by ishaanbhimwal**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/geany-themes/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+short_desc="Collection of color schemes for Geany"
+maintainer="Ishaan Bhimwal <ishaanbhimwal@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://git.geany.org/geany-themes/"
+distfiles="https://git.geany.org/geany-themes/snapshot/geany-themes-${version}.tar.gz"
+checksum=7e852f8cfde940704282de8554c6de4a36d1f6a2f5641b4c24d8eb537275a7cc
+
+do_install() {
+	vmkdir /usr/share/geany/colorschemes
+	vcopy colorschemes/* /usr/share/geany/colorschemes
```
Comment (classabbyamp) [inline srcpkgs/geany-themes/template:15]:
```suggestion
	vmkdir /usr/share/geany
	vcopy colorschemes /usr/share/geany
```

**Comment by ishaanbhimwal** [comment]
i hate squashing commits. if you are okay can I open a new pr. maybe after sometime...

**Comment by classabbyamp** [comment]
to avoid squashing you can use `git commit --amend`

**Comment by ishaanbhimwal** [comment]
fixed finally...thanks to https://stackoverflow.com/a/5189600. 

```
git rebase -i HEAD~n
git commit --amend
```

will make a new PR after little more testing. https://github.com/ishaanbhimwal/void-packages/commit/8b3dff480100cd686d9147f9e668380f317f2e4d

**Pull request closed without merging**